.github/workflows/publish-bugfix-release.yml

```diff
@@ -91,6 +93,8 @@ jobs:
         run: |
           git add gradle.properties $(find charts -name Chart.yaml) $(find charts -name README.md)
           git commit --message "Prepare release ${{ inputs.version }}"
+          
+          git push origin ${{ github.ref_name }}
```

Comment:
```suggestion
          git push
```

Reply:
Wasn't sure just the push would work fine so decided to pass the `ref_name`. Then I wasn't sure the branch would be discoverable and then added the `fetch-depth: 0`.

Nevertheless I tested this in a private repo and the simpler version works, so i'll change.